ddtrace/appsec/_iast/__init__.py

```diff
@@ -91,6 +93,26 @@ def enable_iast_propagation():
     _iast_propagation_enabled = True
 
 
+def _iast_pytest_activation():
```

Comment:
This function is kind of confusing because it seems like `oce.reconfigure` is relying on a bunch of external state set up ahead of time. Fixing that is probably out of the scope of this change but it's worth mentioning.

Reply:
Indeed, the thing is that we're reconfiguring all the IAST machinery after its start, changing mainly the default sampling values (to disable them in this case). These values are currently only read from env vars, so that's the reason for this code.